stories/image.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { storiesOf } from '@kadira/storybook';
-import Image from '../src/image';
+import Image from '../lib/Image';
```

Comment:
now storybook use the compiled files, instead of using the pre-compiled versions